src/main/java/dev/j3fftw/litexpansion/machine/AdvancedSolarPanel.java

```diff
@@ -141,22 +150,39 @@ public int getCapacity() {
     @AllArgsConstructor(access = AccessLevel.PRIVATE)
     public enum Type {
 
-        ADVANCED(Items.ADVANCED_SOLAR_PANEL, ADVANCED_DAY_RATE, ADVANCED_NIGHT_RATE, ADVANCED_OUTPUT,
-            ADVANCED_STORAGE, new ItemStack[] {
+        BASIC(Items.BASIC_SOLAR_PANEL, 10, 1, 10_000, new ItemStack[] {
```

Comment:
Is this what you mean by 'you should rather add these values to the corresponding enum as fields instead'? @TheBusyBiscuit